### PR TITLE
doc: Updates for BT5 and Mesh

### PR DESCRIPTION
**Pull request by rymanluk**

### CODE REVIEW

File: docs/network/ble/ble_intro.md [hunk 1]
```diff
@@ -25,6 +24,16 @@ NimBLE complies with Bluetooth Core Specification 5.0 which makes it an ideal wi
 
 The Bluetooth 5 is backward compatible with previous Bluetooth version 4.2 which is also supported by Apache Mynewt.
 
+### Bluetooth Mesh features
+
+Nimble supports Bluetooth Mesh which is great addition to IoT connectivity space and opens wide range of new possibilities for it.
```
Comment (cwanda) [inline docs/network/ble/ble_intro.md]: Maybe reword:
"NimBLE supports Bluetooth Mesh. Bluetooth Mesh opens a wide range of new possibilities for the IoT connectivity space.  NimBLE fully supports the following Bluetooth Mesh features:

File: custom-theme/pages/ble.html [hunk 2]
```diff
@@ -42,6 +47,15 @@
           <li>Transmit Power Service</li>
         </ul>
 
+        <h3>Bluetooth Mesh support</h3>
+        <ul>
+          <li> Advertising and GATT beares </li>
```
Comment (cwanda) [inline custom-theme/pages/ble.html]: typo: "beares"->"bearers"

File: docs/network/ble/ble_intro.md [hunk 3]
```diff
@@ -51,5 +60,6 @@ Mynewt comes with two built-in projects that allow users to play with NimBLE, tr
 
 1. **bletiny** : A simple shell application which provides a basic interface to the host-side of the BLE stack. 
 2. **bleprph**: A basic peripheral device with no user interface. It advertises automatically on startup, and resumes advertising whenever a connection is terminated. It supports a maximum of one connection.
+3. **blemesh**: Sample application for BLE Mesh node with on/off model.
```
Comment (cwanda) [inline docs/network/ble/ble_intro.md]: Change "Sample application" -> "A sample application" to make the list consistent. 
I am not familiar with BLE Mesh nodes. The literature indicates a server/client model. Does this application implement a server, client, or both?

File: docs/network/ble/ble_intro.md (hunk 1 above)
Comment (rymanluk) [reply docs/network/ble/ble_intro.md]: agree - will fix

File: docs/network/ble/ble_intro.md (hunk 3 above)
Comment (rymanluk) [reply docs/network/ble/ble_intro.md]: "Node" defines that.
